common/sensor/sensor.c

```diff
@@ -250,6 +257,11 @@ void sensor_poll_handler(void *arug0, void *arug1, void *arug2)
 
 			k_yield();
 		}
+
+		if (is_sensor_ready_flag == false) {
+			is_sensor_ready_flag = true;
+		}
```

Comment:
If this is false, it sets it to true.
If this is already true, then it's true.

This can probably just be replaced with:
`s_sensor_ready_flag = true;`